broker/ztf_archive/_download_data.py

```diff
@@ -97,17 +101,17 @@ def _download_alerts_file(file_name, out_dir, block_size, verbose):
             data.extractall(out_dir)
 
 
-def download_data_date(year, month, day, block_size=1024, verbose=True):
+def download_data_date(year, month, day, block_size=1024, verbose=True) -> None:
```

Comment:
Is there a reason these args don't have a type specified? This syntax is new to me, but based on the other functions I think this should be:

```
def download_data_date(
        year: int, 
        month: int, 
        day: int, 
        block_size: int = 1024, 
        verbose: bool = True) -> None:
```

Reply:
Correct, I must have just missed those.